pkg/policy/types/policyentry.go

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package types
+
+import (
+	"github.com/cilium/cilium/pkg/labels"
+	"github.com/cilium/cilium/pkg/policy/api"
+)
+
+// PolicyEntry specifies the L3/L4 details of a single policy rule
+//
+// +deepequal-gen=true
+// +deepequal-gen:private-method=true
+type PolicyEntry struct {
+	// Authentication specifies the cryptographic authentication required for the traffic to be allowed
+	Authentication *api.Authentication
+
+	// Log specifies custom policy-specific Hubble logging configuration.
+	Log api.LogConfig
+
+	// EndpointSelector specifies the endpoint that this rule applies to
+	EndpointSelector api.EndpointSelector
```

Comment:
nit, but we can defer to a separate PR: I think clarity could benefit from referring to this as the Subject.
```suggestion
	// Subject specifies the endpoint that this rule applies to
	Subject api.EndpointSelector
```

Reply:
I like this suggestion. I have added a new commit to this PR that changes the name from EndpointSelector to Subject.